contracts/aave/AaveV2Contract.sol

```diff
@@ -0,0 +1,52 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity 0.8.15;
+
+import "../interfaces/IPoolV3.sol";
+import "../interfaces/LiquidityPoolInterfaces.sol";
+import "../interfaces/IPoolAddressesProvider.sol";
+import "hardhat/console.sol";
+import "../libraries/PercentageMath.sol";
+import "@chainlink/contracts/src/v0.8/interfaces/AggregatorV3Interface.sol";
+
+contract AaveWrapper {
+    using PercentageMath for uint256;
+    struct DEPOSITS {
+        uint256 amount;
+        uint256 price;
+    }
+    AggregatorV3Interface internal priceFeed;
+    mapping(address => mapping(string => DEPOSITS)) public deposits;
+
+    IPoolV2 public constant POOL =
+        IPoolV2(address(0x7d2768dE32b0b80b7a3454c06BdAc94A69DDc7A9));
```

Comment:
i think making this address constant would be a good idea rather than using them like this.

Reply:
we were just reserving extra 32 bytes for constant instead now we are passing constant value directly